android/app/src/androidTest/java/com/trustwallet/core/app/utils/TestPrivateKey.kt

```diff
@@ -37,17 +37,19 @@ class TestPrivateKey {
     @Test
     fun isValidForInvalidData() {
         val bytes = byteArrayOf(0xde.toByte(), 0xad.toByte(), 0xbe.toByte(), 0xef.toByte())
-        assertFalse(PrivateKey.isValid(bytes))
+        assertFalse(PrivateKey.isValid(bytes, Curve.SECP256K1))
+        assertFalse(PrivateKey.isValid(bytes, Curve.ED25519))
     }
 
     @Test
     fun isValidForValidData() {
```

Comment:
did we have a duplication for Java/Kotlin testing?

Reply:
added